config/settings/test.py

```diff
@@ -91,6 +91,9 @@
 SALESFORCE_DOCSET_STATUS_INACTIVE = "Inactive"
 SALESFORCE_DOCSET_RELATION_FIELD = SALESFORCE_DOCSET_SOBJECT
 
+# this will slow things down and should only be used for testing
+CACHE_VALIDATION_MODE = env("CACHE_VALIDATION_MODE", default=False)
```

Comment:
No need to duplicate this line between the various settings files. Just put it like this in base.py

Reply:
This project used a different pattern and I don't know why. I just copied it. I can either do this one in the "right way" and clean the rest up later or I can just clean them all up at the same time later.

Reply:
Okay, fair enough. If it's consistent with other settings I have less of a problem with it.

Reply:
+1 to cleaning the settings up...